comfy_cli/command/generate/app.py

```
@@ -298,6 +308,78 @@ def _refresh() -> None:
+    flag_by_name = {f.name: f for f in flags}
+    for name, value in list(values.items()):
+        flag = flag_by_name.get(name)
+        if flag is None or flag.upload_mode is None or not isinstance(value, str):
+            continue
+        if value.startswith(("http://", "https://", "data:")):
+            continue
+        path = Path(value).expanduser()
+        if not path.is_file():
+            continue
+        if flag.upload_mode == "base64":
+            import base64 as _base64
```

Pylint (4.0.5)

[convention] Import outside toplevel (base64)

(C0415)